webapp/app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -102,7 +102,7 @@ msgstr "Nutzungsbedingungen - Der Steuerlotse für Rente und Pension"
 
 #: app/routes.py:270
 msgid "about.header-title"
-msgstr "Projektinfos - Der Steuerlotse für Rente und Pension"
+msgstr "Impressum - Der Steuerlotse für Rente und Pension"
```

Comment:
I am also wondering if we have to change this?

Reply:
That was not on purpose. Something must have gotten mixed up in the merge conflict.